test/sharness/t0010-basic-commands.sh

```diff
@@ -22,7 +22,7 @@ test_expect_success "ipfs version output looks good" '
 '
 
 test_expect_success "ipfs help succeeds" '
-	ipfs help >help.txt
+	ipfs -h >help.txt
```

Comment:
I think that both `ipfs help` and `ipfs -h` should be valid for printing out the help text.